src/Table/index.stories-hidden.js

```diff
@@ -0,0 +1,225 @@
+/**
+ * TO VIEW THIS STORY LOCALLY:
+ * change the file name from `.stories-hidden.js` to `.stories.js`.
+ *
+ * This component is still in development, so we can't publish the docs publicly yet.
+ */
+
+/* eslint-disable jsx-a11y/anchor-is-valid,react/jsx-key */
+import React from "react";
+import Table from "./";
+import MenuButton from "../MenuButton";
+import Checkbox from "../Checkbox";
+
+export const Overview = () => (
+  <div className="padding--all bgColor--blueGrey">
+    <div className="margin--bottom--xl">
+      <Table
+        colVisibility={["*", "m", "*", "l"]}
+        colLayout={{
+          s: "invalid lol", // falls back to even 1fr cols
```

Comment:
invalid CSS values passed here will fall back to something sensible and trigger a `console.warn`.